data/Abhish7k.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "Abhishek",
+  "bio": "Full Stack Developer",
+  "links": [
+    {
```

Please add the `name` key of the link. For `links` it's necessary to use `name` key. 

```suggestion
  "links": [
   {
  "name": "Follow me on GitHub",
```

 Or modify it to use as socials on your profile

```suggestion
  "socials": [
    {
```